pinot-common/pom.xml

```diff
@@ -46,6 +46,8 @@
         <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-surefire-plugin</artifactId>
         <configuration combine.children="override">
+          <forkCount>1</forkCount>
```

Comment:
This is also the default value: https://maven.apache.org/surefire/maven-surefire-plugin/test-mojo.html#forkCount

See also https://maven.apache.org/surefire/maven-surefire-plugin/examples/fork-options-and-parallel-execution.html#forked-test-execution

Reply:
Yes, I override the parent settings with defaults.